lib/rubocop/config_path_finder.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require_relative 'file_finder'
+
+module RuboCop
+  # This class has methods related to finding configuration path.
+  # @api private
+  class ConfigPathFinder
```

Comment:
Perhaps just `ConfigFinder`?

Reply:
Thanks for the suggestion! I've update this class name.